### PR TITLE
implement works and about myself section in the design

### PR DESCRIPTION
**Pull request by kennedyowusu**
1. Created a works section
2. Created about myself section
3. Added the CSS codes to match the design
4. Used CSS flexbox and grid to implement these designs

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -87,3 +86,299 @@ <h4 class="sub-title">I'm a software developer</h4>
...
+
+          <div class="about-me-text-container">
+            <p class="about-me-text">
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </p>
+          </div>
+        </div>
+
+        <!-- <div class="about-me-text-container"></div> -->
```
Comment (cynthiainga) [inline index.html]: - [ ] I suggest you to put all this code that you added in the `body` and `html` tag for HTML best practice , it's a part of your HTML code is supposed to be inside and not outside the `html` and `body` tag 🙃 😉

File: index.html [hunk 2]
```diff
@@ -87,3 +86,299 @@ <h4 class="sub-title">I'm a software developer</h4>
...
 </html>
+
+
+      <!-- My works begins here -->
+
+      <section class="my-works">
+        <h2 class="section-title">
+          My Recent Works
+          <hr class="horizontal-line" />
+        </h2>
```
Comment (cynthiainga) [inline index.html:93]: - [ ] As required in the project requirements you should make sure your design stick as much as possible to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27). You should make sure your title has the same font size as in the Figma design, it;s 40px 🙃 🙃
- [ ] As in the project requirements you must stick to the design as much as possible (e.g., font, colors, images, text, margins)

Your design

Screen Shot 2022-10-25 at 21 04 35

Figma design

Screen Shot 2022-10-25 at 21 04 11

File: index.html [hunk 3]
```diff
@@ -87,3 +86,299 @@ <h4 class="sub-title">I'm a software developer</h4>
...
+                    standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.
+                  </p>
+                  <ul class="card-list">
+                    <li class="card-list-item">css</li>
+                    <li class="card-list-item">html</li>
+                    <li class="card-list-item">bootstrap</li>
+                    <li class="card-list-item">Ruby</li>
+                  </ul>
+                  <a class="card-link" href="#">See Project</a>
```
Comment (cynthiainga) [inline index.html:118]: - [ ] The buttons should follow the button styling: Active, Hover, and Pressed. Please kindly add it for all the buttons 🙃 😉 

Screen Shot 2022-10-25 at 21 02 24

File: index.html [hunk 4]
```diff
@@ -87,3 +86,299 @@ <h4 class="sub-title">I'm a software developer</h4>
...
+                </div>
+              </div>
+            </li>
+
+            <li>
+              <div class="second-card">
```
Comment (cynthiainga) [inline index.html:124]: - [ ] As in the project requirements you must stick to the design as much as possible (e.g., font, colors, images, text, margins). Kindly make sure the space between your word is the same, and the fonts(font-size, font-family, font-weight, font-style,...) are the same🙃 🙃 

Your design

Screen Shot 2022-10-25 at 21 12 17

Figma design

Screen Shot 2022-10-25 at 21 12 38

File: index.html [hunk 5]
```diff
@@ -87,3 +86,299 @@ <h4 class="sub-title">I'm a software developer</h4>
...
+
+              <li class="about-me-list-item">
+                <h3>Gitlab</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3 style="margin-bottom: 100px">Terminal</h3>
+              </li>
+            </ul>
+          </div>
```
Comment (cynthiainga) [inline index.html:369]: - [ ] Please you should make sure that your design stick as much as possible to the Figma design, that means (font, margin, padding, color,...). You should make sure that the space between your languages, skills, and framework is the same as in the Figma, also the fonts 🙃 😉 

Your design

Screen Shot 2022-10-25 at 20 43 56

Figma design

Screen Shot 2022-10-25 at 20 44 22

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @kennedyowusu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] All linters are passing ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Github flow ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [ ] Please kindly make your README file professional and documented, you can use this [README templave](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse and then put the important pieces of information inside 👍 🙃
- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (kennedyowusu) [reply index.html:118]: @cynthiainga I don't understand this comment. Taking a look at the buttons in the Figma design, the buttons are either bordered with a color (1px solid #8993A4) which I used, a button with a background color (#FF6B00) which I used, or a button with (background: rgba(255, 255, 255, 0.24)) which I used also. So your feedback on the button is very confusing. Please do come again on that and add images where possible from the figma design to make clear.

Comment (cynthiainga) [reply index.html:118]: Hi @kennedyowusu, the styling: Active, Hover, and Pressed means that when for example users press the buttons change the color, and have the color of the pressed button, or when the button is disabled it takes the color of the disabled button and so one... I think it's clear now 🙃

File: index.html (hunk 1 above)
Comment (kennedyowusu) [reply index.html]: Done @cynthiainga

File: index.html (hunk 2 above)
Comment (kennedyowusu) [reply index.html:93]: Done @cynthiainga

File: index.html (hunk 5 above)
Comment (kennedyowusu) [reply index.html:369]: Please I have resolved these. Kindly check again

**Comment by kennedyowusu** [comment]
@cynthiainga I have resolved the issues you identified. Kindly check again and give feedback so I don't have to request for another review, please. Thanks and I will be waiting on you

File: index.html (hunk 3 above)
Comment (kennedyowusu) [reply index.html:118]: I have understood it now and I **have resolved it**

**Comment by kennedyowusu** [comment]
@cynthiainga Please kindly check again and get back on the issues. I have resolved all of them to the best of my knowledge.

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html:93]: Fixed.

File: index.html [hunk 6]
```diff
@@ -84,6 +83,304 @@ <h4 class="sub-title">I'm a software developer</h4>
...
+                    standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.
+                  </p>
+                  <ul class="card-list">
+                    <li class="card-list-item">css</li>
+                    <li class="card-list-item">html</li>
+                    <li class="card-list-item">bootstrap</li>
+                    <li class="card-list-item">Ruby</li>
+                  </ul>
+                  <a class="card-link" href="#">See Project</a>
```
Comment (Abenezer-Tilahun) [inline index.html:118]: - [ Optional ] Good Job for implementing the previous reviewer suggestion but you implemented the the styling: Active, Hover, and Pressed for card-list(html,css etc) not for button "See Project", so kindly work for it.

File: index.html (hunk 1 above)
Comment (kennedyowusu) [reply index.html]: All issues raised have been fixed please @cynthiainga and @Abenezer-Tilahun

File: index.html (hunk 2 above)
Comment (kennedyowusu) [reply index.html:93]: @Abenezer-Tilahun and @cynthiainga I have resolved this also.

File: index.html (hunk 4 above)
Comment (Abenezer-Tilahun) [reply index.html:124]: Fixed.

File: index.html (hunk 5 above)
Comment (Abenezer-Tilahun) [reply index.html:369]: Fixed

File: index.html (hunk 6 above)
Comment (kennedyowusu) [reply index.html:118]: @Abenezer-Tilahun, should it be on both or on only the see project?

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @kennedyowusu  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (Abenezer-Tilahun) [reply index.html:118]: @kennedyowusu from the Figma design it's better to implemented for only the button "See Project".

Comment (kennedyowusu) [reply index.html:118]: Thank you. I have resolved it. @Abenezer-Tilahun

**Pull request merged**